Examples/PatchExample.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace QModManager.Examples
+{
+    using QModManager.API.ModLoading;
+
+    [QModCoreInfo("QExampleMod", "Example Mod", "QModManager Dev Team", API.QModGame.Subnautica)]
+    [QModLoadBefore("SomeMod")]
+    [QModLoadBefore("SomeMod2")]
+    [QModLoadAfter("SomeOtherMod")]
+    [QModDependency("ApiMod")]
+    [QModDependency("OtherAPIMod", 2, 2, 1)]
+    internal static class PatchExample
+    {
+        [QModPrePatchMethod]
+        public static PatchResults PatchMyModEarly()
+        {
+            // Early mod patching happens here
+            return PatchResults.OK;
```

Comment:
Is `PatchResults` only available for `[QModPrePatchMethod]`?

Reply:
No. There is no restriction in place for that.  
It's just difficult to show an optional return value with a single example.

Reply:
People should know that this works with all methods. Maybe a comment?